iron-scroll-manager.html

```diff
@@ -62,6 +64,17 @@
       return this._lockingElements[this._lockingElements.length - 1];
     },
 
+    /**
+     * The current element that defines the DOM boundaries of the
+     * scroll lock. This is always the most recently locking element.
+     * Required for modulizer
+     *
+     * @return {!Node|undefined}
+     */
+    getCurrentLockingElement: function() {
```

Comment:
instead of adding this, we could convert `currentLockingElement` to be just a property (not a getter)
```js
    /**
      * The current element that defines the DOM boundaries of the
      * scroll lock. This is always the most recently locking element.
      *
      * @type {!Node|undefined}
      */
     currentLockingElement: undefined,
```

and update it in `pushScrollLock ` and `removeScrollLock`
```js
this.currentLockingElement = this._lockingElements[this._lockingElements.length - 1];
```